### DISCUSSION TITLE
A series in-depth blogs for GraphScope

### DISCUSSION THREAD

**Discussion by siyuan0322** [alibaba/GraphScope#1175] [Ideas]
It's maybe helpful to have a series of in-depth blogposts targeting at various aspect for user who want to be more familiar with graphscope and even contribute to graphscope.

Here's a list of very rough ideas:

1. The getting started guide.
2. Interpretation of architectures of different deployment 
3. The procedure of loading datasets.
4. The dynamic generation, compilation and execution of applications
5. The selector.
6. Detailed information of Graphs.
7. Execution of operations.
8. Log system for all components (gae, gie, gle), for the ease of debugging.
9. How do we achieve zero copy data sharing via Vineyard
10. GIE related stuff (I'm not very familiar with this part)
11. E2E testing guide.
12. Networkx module itself deserves several blog posts.
13. GLE related stuff (I'm not familiar with this too, but this is a large topic)
14. GraphScope Store related stuff.

**Comment by imbajin** [comment]
A nice plan~

And the priority of the ideas can be sorted like : 

```bash
       													  							 --> the loading module
use/debug first --> overall arch --> the context of graph computation / tp traverser --> the storage module (v6d/maxgraph) 
														   							 --> the GIE/GAE/GXE module..
```

Another way to prioritize is to sort according to the needs of internal and external users

**Reply by siyuan0322** [reply]
That's an awesome idea, I almost forgot the using/debugging part :joy: Thank you!
